odoo/addons/base/tests/test_mail.py

```diff
@@ -9,12 +9,13 @@
 from odoo.addons.base.models.ir_qweb_fields import nl2br_enclose
 from odoo.tests import tagged
 from odoo.tests.common import BaseCase
-from odoo.tools import (
+from odoo.tools import misc
```

Comment:
This one is only used for `html_escape`.  Please add a proper import of that function instead.